conf/test_full.config

```diff
@@ -15,7 +15,7 @@ params {
     config_profile_description = 'Full test dataset to check pipeline function'
 
     genome = 'GRCh38'
-    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/cutandrun/samplesheet/test-GSE145187-all.csv'
+    input = 'https://raw.githubusercontent.com/luslab/test-datasets/cutandrun/samplesheet/test-GSE145187-all.csv'
```

Comment:
shouldn't that be on the nf-core repo?

Reply:
Yes I need to switch these over but i cant until the PR is ready to merge as all the tests will break for anyone running the current version